ethnographiques-org.csl

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="expanded" default-locale="fr-FR">
+  <info>
+    <title>Ethnographiques.org (French)</title>
+    <id>http://www.zotero.org/styles/ethnographiques-org</id>
+    <link href="http://www.zotero.org/styles/ethnographiques-org" rel="self"/>
+    <link href="http://www.zotero.org/styles/metropolitiques" rel="template"/>
+    <link href="http://www.metropolitiques.eu/Charte-editoriale,6.html" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
+    <issn>1961-9162</issn>
+    <updated>2021-05-26T13:58:06+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="ordinal-01">ère</term>
+      <term name="ordinal-02">e</term>
+      <term name="ordinal-03">e</term>
+      <term name="ordinal-04">e</term>
+      <term name="cited">op. cit.</term>
+      <term name="page" form="short">p.</term>
+      <term name="issue" form="short">n°</term>
+    </terms>
+  </locale>
+  <macro name="author">
```

Comment:
Any reason not to add proper substitute here instead of the choose (and also include the title -- see the webpage citation in the auto-preview)

Reply:
Reason is I didn't look at that part as the guidelines are a bit messy. ha.
Will fix. Thanks for checking. :)